### PR TITLE
Portals Popover component

### PR DESCRIPTION
**Pull request by fayazara**
# Pull Request Template

## Description

Screenshot2022-07-18 at 01 50 01@2x

A popover for switching portals in helpcenter.

Fixes https://github.com/chatwoot/product/issues/468

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by fayazara** [comment]
> Note

1. Clicking on the portal switch should update a variable the respective selected switch components data - i.e set the `selectedPortal` with portal id/slug/name and locale.

**Review by muhsin-k — COMMENTED** [review]
This component only for help center. Let's keep in `app/javascript/dashboard/routes/dashboard/helpcenter/components` folder.

File: app/javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="portal-popover__container">
+    <header>
+      <div class="actions">
+        <h2>{{ $t('HELP_CENTER.PORTAL.POPOVER.TITLE') }}</h2>
+        <router-link to="#" class="new-popover-link">
```
Comment (nithindavid) [inline app/javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue:6]: We can use `a` tag and reuse `button smooth secondary` classes here - `app/javascript/dashboard/components/layout/sidebarComponents/PrimaryNavItem.vue`

File: app/javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="portal-popover__container">
+    <header>
+      <div class="actions">
+        <h2>{{ $t('HELP_CENTER.PORTAL.POPOVER.TITLE') }}</h2>
```
Comment (nithindavid) [inline app/javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue:5]: Reuse heading typography classes here.

**Review by nithindavid — CHANGES_REQUESTED** [review]
Please see the comments

**Comment by nithindavid** [comment]
@fayazara We can rename the component `javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue` => `app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalPopover`

**Comment by fayazara** [comment]
> @fayazara We can rename the component `javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue` => `app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalPopover`

Okat, moving all these components with to the new folder structure.

File: app/javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue (hunk 1 above)
Comment (fayazara) [reply app/javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue:6]: using the a tag will refresh the page. @nithindavid

**Review by nithindavid — APPROVED** [review]

**Review by iamsivin — APPROVED** [review]

**Pull request merged**